src/analytics/docs.ts

```diff
@@ -606,7 +606,7 @@ export const eventDocs: Record<AnalyticsEventType, string> = {
   [EarnEvents.earn_withdraw_add_gas_press]: `When the user doesn't have enough for gas and clicks on the button to add gas token`,
   [EarnEvents.earn_info_learn_press]: `When the user taps 'Learn More' on the earn info page`,
   [EarnEvents.earn_info_earn_press]: `When the user taps 'Start Earning' on the earn info page `,
-  [EarnEvents.earn_active_pools_cta_press]: `When the user taps one of the CTAs in the active pools card in the discover tab. The action property denotes the CTA, either myPools or exploreOpenPools`,
+  [EarnEvents.earn_active_pools_cta_press]: `When the user taps one of the CTAs in the active pools card in the discover tab.`,
```

Comment:
```suggestion
  [EarnEvents.earn_active_pools_cta_press]: `When the user taps on the active pools card in the discover tab.`,
```

Reply:
or maybe we just have a new event? it's not really a cta press anymore it's just active pool card press

Reply:
Fixed in 21174eee2f43974eae2b8ae98569fc42a6a858e6!